IntrepidSwiftWisdom.podspec

```diff
@@ -17,6 +17,6 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |cs|
     cs.source_files = "SwiftWisdom/Core/**/**/*.swift"
-    cs.dependency 'IP-UIKit-Wisdom', '0.0.9'
+    cs.dependency 'IP-UIKit-Wisdom', '0.0.10'
```

Comment:
Actually, @brightredchilli, if we bump this, we should also bump the IntrepidSwiftWisdom.podspec version, theoretically to 0.7.3...? Since we are no longer really maintaining IntrepidSwiftWisdom, should I just revert this?